### DISCUSSION TITLE
Use Ideal in PaymentSheet

### DISCUSSION THREAD

**Discussion by alexlipa91** [flutter-stripe/flutter_stripe#1198] [Q&A]
When I create a PaymentIntent using the api and adding `ideal` as payment method I don't get ideal within the sheet but only cards

I would like to know if it is possible to use ideal in this flow or I have to use Checkout and an external browser

**Comment by remonh87** [comment]
Yes this is possible I use it myself for my projects.  You need to make sure the payment method is enabled in the stripe dashboard. If this is the case I would recommend to check the stripe dashboard and see the status of the payment intents.  Also we wrote an elaborate guide on payment sheets: https://docs.page/flutter-stripe/flutter_stripe/ideal
